cli/main/main.go

```diff
@@ -11,29 +11,38 @@ import (
 )
 
 func main() {
-	if len(os.Args) < 3 {
-		fmt.Println("Usage: go run cli/main/main.go <instagram|twitter> <username>")
-		return
-	}
+	var platformArg string
+	var userNameArg string
 
-	platformArg := os.Args[1]
-	userNameArg := os.Args[2]
+	kafkaAddress := utils.GetStringFromEnvWithDefault("KAFKA_ADDRESS", "my-kafka:9092")
+	instagramTopic := utils.GetStringFromEnvWithDefault("INSTAGRAM_TOPIC", "user_names")
+	twitterTopic := utils.GetStringFromEnvWithDefault("TWITTER_TOPIC", "twitter-user_names")
```

Comment:
why not only have one `KAFKA_INSERT_TOPIC`?

Reply:
I would like to leave it like this for now because it makes the combination of local testing and Kubernetes deployment more convenient and cleaner. Replacing both variables by `KAFKA_INSERT_TOPIC` could make the usage more easier but would mean additional code to make it running on both enviroments